src/app/functions/server/onerep.ts

```diff
@@ -216,6 +219,28 @@ export async function optoutProfile(profileId: number): Promise<void> {
     );
   }
 }
+
+export async function activateAndOptoutProfile(
+  profileId: number,
+): Promise<void> {
+  try {
+    const { status: profileStatus } = await getProfile(profileId);
```

Comment:
Hmm, so this means we're doing a call to OneRep every time we load the dashboard for Premium users. Is OneRep OK with that? And isn't this going to delay our serving the dashboard quite a bit?

Reply:
That’s a good point. We might need to consider either adding the profile status to our `onerep_profiles` table or I can also just move the `hasInitialScan` check up and return early.

Reply:
I think [ff91bc7](https://github.com/mozilla/blurts-server/pull/3705/commits/ff91bc7bb832edc5449bd8ea15e346115aa54998) should already avoid the repeated calls to OneRep.